integration-tests/gradle/src/test/java/io/quarkus/gradle/TestFixtureModuleWithPackagePrivateParentTest.java

```diff
@@ -6,12 +6,11 @@
 import java.io.IOException;
 import java.net.URISyntaxException;
 
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 public class TestFixtureModuleWithPackagePrivateParentTest extends QuarkusGradleWrapperTestBase {
 
-    @Disabled("See https://github.com/quarkusio/quarkus/issues/47760")
+    //@Disabled("See https://github.com/quarkusio/quarkus/issues/47760")
```

Comment:
Can we delete the line entirely, instead of commenting it out?

Reply:
Sure, I'll delete it in a follow up.